tests/integration/test_prometheus_scrape_multiunit.py

```diff
@@ -48,13 +48,14 @@
 idle_period = 90
 
 
+@pytest.mark.skip(reason="xfail")
```

Comment:
this feels bad. 'xfail' is not a reason for skipping a test.
1) why are we skipping it instead of removing it altogether?
2) should we be xfailing it instead? 

Also applies below

Reply:
I can remove the change from this PR if you want, to keep it cleaner, but imho `xfail` is a reason for skipping a test; if we don't care about the test passing, isn't it running it a waste of time?

I don't want to remove it because if we rewrite tests, it could be useful to have them there, but the `skip(reason="we were xfailing this")` sounds like a good reason to me

However if you disagree, I'll remove it and we'll talk about it in more detail